protocol/tests/coordinator_test.go

```diff
@@ -24,10 +26,16 @@ func testConfig() common.Config {
 }
 
 func TestRPCBasic(t *testing.T) {
-	s, err := server.NewServer("test", testAddr, true, testConfig(), nil, 5, time.Hour)
+	s, err := server.NewServer("test", testAddr, testConfig(), nil, 5, time.Hour)
 	if err != nil {
 		t.Errorf("Error creating new server")
 	}
+	testAddrTCP, _ := net.ResolveTCPAddr("tcp4", testAddr)
+	l, err := net.ListenTCP("tcp4", testAddrTCP)
```

Comment:
Should close the listener

Reply:
The server above being closed may close it, per https://golang.org/src/net/http/server.go?s=82330:82376#L2668

adding an explicit close as well